gateway/1.4/modules/ROOT/pages/index.adoc

```diff
@@ -72,78 +74,32 @@ The following table summarizes the differences between Connected Mode and Local
 | | Connected Mode | Local Mode
 
 | *Use Case*
-| Choose Connected Mode for centralized management, observability, and security. Flex Gateway in Connected mode connects to the control plane.
-| Choose Local Mode to operate independently of the control plane in a mostly disconnected manner. Manage Flex Gateway in Local Mode with locally stored declarative configuration files. Use Local Mode to build CI/CD pipelines.
+| Choose for centralized management, observability, and security. Flex Gateway in Connected mode connects to the control plane.
```

Comment:
You could tighten by removing the Choose and Apply statements, since those are covered in the table row heading.

Reply:
Connected Mode (note capitalization)

Reply:
Fixed